backend/tests/api-e2e/test_api_uq.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+from flask import current_app as app
+
+from app.infrastructure import database
+from app.legacy.core.projects import get_projects_addresses
+from tests.conftest import UserAccount, Client
+from tests.helpers.constants import STARTING_EPOCH
+
+
+@pytest.mark.api
+def test_uq_for_user(client: Client, ua_alice: UserAccount):
+    client.move_to_next_epoch(STARTING_EPOCH + 1)
+    client.move_to_next_epoch(STARTING_EPOCH + 2)
+    client.move_to_next_epoch(STARTING_EPOCH + 3)
+
+    epoch_no = client.wait_for_sync(STARTING_EPOCH + 3)
+    app.logger.debug(f"indexed epoch: {epoch_no}")
+
+    USER_NOT_FOUND = 404
+    _, code = client.get_user_uq(ua_alice.address, 4)
+    assert code == USER_NOT_FOUND
+
+    database.user.add_user(ua_alice.address)
+
+    res, code = client.get_user_uq(ua_alice.address, 4)
+    assert code == 200
+    assert res["uniquenessQuotient"] == "1.0" or res["uniquenessQuotient"] == "0.2"
```

Comment:
Would using constant from "app" make sense here? If we ever change 0.2 -> 0.3 we will probably catch it at the PR pipeline rather than locally I think - so not sure if hardcoding make sense. But also I don't have strong feelings about doing it one way or the other.